Dockerfile

```
@@ -1,15 +1,17 @@
-FROM quay.io/ukhomeofficedigital/hocs-base-image as builder
+FROM quay.io/ukhomeofficedigital/hocs-base-image-build as builder
```

This needs to be the libre-office one? 

```suggestion
FROM quay.io/ukhomeofficedigital/hocs-libreoffice-build as builder
```